datacenter/ucp/2.2/guides/admin/configure/external-auth/index.md

```diff
@@ -117,7 +124,7 @@ user which can still login with the recovery admin password.
 UCP saves a minimum amount of user data required to operate. This includes
 the value of the username and full name attributes that you have specified in
 the configuration as well as the distinguished name of each synced user.
-UCP does not query, or store any additional data from the directory server.
+UCP does not query or store any additional data from the directory server.
```

Comment:
Maybe just say "UCP does not store any additional data...", since the UCP will query the LDAP server on every LDAP sync (every 30 mins by default).